sklearn/neighbors/classification.py

```diff
@@ -12,10 +12,10 @@
 from scipy import stats
 from ..utils.extmath import weighted_mode
 
-from .base import \
-    _check_weights, _get_weights, \
-    NeighborsBase, KNeighborsMixin,\
-    RadiusNeighborsMixin, SupervisedIntegerMixin
+from .base import (
+        _check_weights, _get_weights,
```

Comment:
Huh? Surely a single tab is correct, and this results in "E126 continuation line over-indented for hanging indent"...?